src/idvc/dvc_interface.py

```diff
@@ -2923,7 +2923,10 @@ def createPointCloud(self, **kwargs):
             self.pointCloud = pointCloud
         else:
             pointCloud = self.pointCloud
-
+        # instead of translating the point cloud so that point0 is in the point cloud
+        # we add point0 to the point cloud.
+        if hasattr(self, 'point0'):
+            pointCloud.SetPoint0(self.point0_world_coords)
```

Comment:
I don't think we should do this. Because then the points aren't evenly spaced and the overlap size around point 0 will not be consistent with the rest of the cloud. E.g. it ends up looking like this:
![image](https://user-images.githubusercontent.com/60604372/205026479-80c9b165-2ee9-4bd1-accd-b5b3618a0d82.png)
And e.g. with an overlap of 0:
![image](https://user-images.githubusercontent.com/60604372/205026664-375e7962-5c84-4632-8f47-002a4027bd32.png)
there's clearly overlap around point 0

Reply:
my understanding is that we had approval from Catherine that this is fine? So re-opening the PR.

Reply:
I need to test this again and re-review